### PR TITLE
cleanup: remove jindofs temporarily

### PR DESCRIPTION
**Pull request by AlbeeSo**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:

Jindofs has not been maintained on CSI for some time. Fuse will eventually need to be run in a containerized way and jindofs needs to be re-adapted in the future.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: build/multi-private/Dockerfile.multi [hunk 1]
```diff
@@ -29,7 +29,6 @@ RUN if [[ $(echo $TARGETPLATFORM | cut -f2 -d '/') == "amd64" ]]; then curl http
 RUN if [[ $(echo $TARGETPLATFORM | cut -f2 -d '/') == "amd64" ]]; then curl http://ack-csiplugin.oss-cn-hangzhou.aliyuncs.com/pre/ossfs/ossfs_${update_ossfsVer}_centos7.0_x86_64.rpm -o /root/ossfs_${update_ossfsVer}_centos7.0_x86_64.rpm; fi
 RUN if [[ $(echo $TARGETPLATFORM | cut -f2 -d '/') == "amd64" ]]; then curl http://ack-csiplugin.oss-cn-hangzhou.aliyuncs.com/pre/ossfs/ossfs_${update_ossfsVer8u}_centos8_x86_64.rpm -o /root/ossfs_${update_ossfsVer8u}_centos8_x86_64.rpm; fi
 
-RUN if [[ $(echo $TARGETPLATFORM | cut -f2 -d '/') == "amd64" ]]; then mv /multi_data/jindofs-fuse-3.7.3-20211207.tar.gz /; tar zvxf /jindofs-fuse-3.7.3-20211207.tar.gz -C /; fi
```
Comment (huww98) [inline build/multi-private/Dockerfile.multi:32]: remove the jindofs-fuse-3.7.3-20211207.tar.gz file from git repo together with this line?

Comment (AlbeeSo) [reply build/multi-private/Dockerfile.multi:32]: done

**Pull request merged**